content/de/_index.html

```diff
@@ -37,7 +37,7 @@
 
 {{< blocks/section id="video" background-image="kub_video_banner_homepage" >}}
 <div class="light-text">
-<h2>Die Herausforderungen bei der Migration von über 150 Microservices auf Kubernetes</h2>
+<h2>Die Herausforderungen bei der Migration is in the über 150 Microservices auf Kubernetes</h2>
```

Comment:
Hi @Shreyar888, This document seems to be in German , are you sure these are the right changes you have made?